tests/Test/WebTestCaseTest.php

```diff
@@ -0,0 +1,281 @@
+            ->will($this->onConsecutiveCalls($repository, $session));
+
+        $cookieJar = $this->getMock('stdClass', ['set']);
+
+        $this->client
+            ->method('getCookieJar')
+            ->willReturn($cookieJar);
+
+        // Call `login` method
+        $method = new \ReflectionMethod($this->mock, 'login');
+        $method->setAccessible(true);
+        $method->invoke($this->mock);
+    }
+
+    public function testGetFile()
+    {
+        // Call `getFile` method
+        $method = new \ReflectionMethod($this->mock, 'getFile');
+        $method->setAccessible(true);
+        $file = $method->invoke($this->mock, 'test', 'text', 'txt', 'text/txt');
+
+        $this->assertInstanceOf(UploadedFile::class, $file);
+    }
+
+    public function testGetImageFile()
+    {
+        // Call `getImageFile` method
+        $method = new \ReflectionMethod($this->mock, 'getImageFile');
+        $method->setAccessible(true);
+        $file = $method->invoke($this->mock);
+
+        $this->assertInstanceOf(UploadedFile::class, $file);
+    }
+
+    public function testGetPdfFile()
+    {
+        // Call `getPdfFile` method
+        $method = new \ReflectionMethod($this->mock, 'getPdfFile');
+        $method->setAccessible(true);
+        $file = $method->invoke($this->mock);
+
+        $this->assertInstanceOf(UploadedFile::class, $file);
+    }
+
+    public function testGetZipFile()
+    {
+        // Call `getZipFile` method
+        $method = new \ReflectionMethod($this->mock, 'getZipFile');
+        $method->setAccessible(true);
+        $file = $method->invoke($this->mock);
+
+        $this->assertInstanceOf(UploadedFile::class, $file);
+    }
+
+    public function testGetTxtFile()
+    {
+        // Call `getTxtFile` method
+        $method = new \ReflectionMethod($this->mock, 'getTxtFile');
+        $method->setAccessible(true);
+        $file = $method->invoke($this->mock);
+
+        $this->assertInstanceOf(UploadedFile::class, $file);
+    }
+
+    public function testLoadFixtureClass()
+    {
+        $loader = $this->getMockBuilder('Symfony\Bridge\Doctrine\DataFixtures\ContainerAwareLoader')
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        // Call `loadFixtureClass` method
+        $method = new \ReflectionMethod($this->mock, 'loadFixtureClass');
+        $method->setAccessible(true);
+        $method->invoke($this->mock, $loader, 'Beelab\\TestBundle\\FakeFixtureDependent');
+
+        $property = new \ReflectionProperty('Beelab\TestBundle\Test\WebTestCase', 'fixture');
+        $property->setAccessible(true);
+        $fixture = $property->getValue($this->mock);
+
+        $this->assertInstanceOf('Beelab\TestBundle\FakeFixtureDependent', $fixture);
+    }
+
+    public function testLoadFixtures()
+    {
+        $this->markTestIncomplete();
+
+        $this->mock
+            ->expects($this->exactly(2))
+            ->method('loadFixtureClass');
```

Comment:
@garak up
Why this does not work?
The mock calls the original loadFixtureClass method making to fail the test.